pkg/entities/offchain_tip_bot_config_notify.go

```diff
@@ -54,7 +54,7 @@ func (e *Entity) ListConfigNotify(guildId string) (rs []response.ConfigNotifyRes
 			}
 
 		} else {
-			totalTx, err = e.repo.OffchainTipBotTransferHistories.GetTotalTransactionByGuildAndToken(guildId, config.Token)
+			_, totalTx, err = e.repo.OffchainTipBotTransferHistories.GetTotalTransactionByGuildAndToken(guildId, config.Token, "")
```

Comment:
why need another var if not used

Reply:
this is from another module that only use the record count